ocf_data_sampler/constants.py

```diff
@@ -0,0 +1,135 @@
+import xarray as xr
+import numpy as np
+
+
+NWP_PROVIDERS = [
+    "ukv",
+    "ecmwf",
+]
+
+
+def _to_data_array(d):
+    return xr.DataArray(
+        [d[k] for k in d.keys()],
+        coords={"channel": [k for k in d.keys()]},
+    ).astype(np.float32)
+
+
+class NWPStatDict(dict):
+    """Custom dictionary class to hold NWP normalizarion stats"""
+
+    def __getitem__(self, key):
+        if key not in NWP_PROVIDERS:
+            raise KeyError(f"{key} is not a supported NWP provider - {NWP_PROVIDERS}")
+        elif key in self.keys():
+            return super().__getitem__(key)
+        else:
+            raise KeyError(
+                f"Values for {key} not yet available in ocf-datapipes {list(self.keys())}"
```

Comment:
Currently we always normalize. The NWP values for different channels have quite a big range, so its good to normalize them

Reply:
Maybe I'm reading it wrong. But doesnt this bit of code ensure that we have put normalizing constants in. If we remove it, we might forget to add normalizing constant it when adding a new NWP?

Reply:
oh yeah no, i meant remove the in providers check (the first if), sorry, wrong code context! Basically if there are no values, it doesn't matter if it's in providers (it shouldn't be, and actually if we get to this point then it is in providers for some reason since the config check didn't catch it), this is not an allowed nwp source. So I think it might be better if it just goes for the values straight away and if they are not there fails hard, instead of popping two different warnings when the provider is not in the list and when the values are not there, if that makes sense?

Tangentially, I think there should be a check that all channels were found in consts, as this can be an issue with accumulated channels getting diffed, but definitely not in this pr; I'l go make a new issue for that